src/ocm/components/clusterConfiguration/CnvCheckbox.tsx

```diff
@@ -31,6 +31,17 @@ export type CnvCheckboxProps = {
   clusterId: string;
 };
 
+const CNVHelperText: React.FC = () => {
```

Comment:
We don't need to define the components as `React.FC` anymore, just as plain functions.
```suggestion
const CNVHelperText = () => {
```